src/SCREENS.ts

```diff
@@ -112,6 +112,7 @@ const SCREENS = {
             CARD_ACTIVATE: 'Settings_Wallet_Card_Activate',
             REPORT_VIRTUAL_CARD_FRAUD: 'Settings_Wallet_ReportVirtualCardFraud',
             CARDS_DIGITAL_DETAILS_UPDATE_ADDRESS: 'Settings_Wallet_Cards_Digital_Details_Update_Address',
+            VERIFY_ACCOUNT: 'Settings_Wallet_Verify_Account',
```

Comment:
Let's add this to `CENTRAL_PANE_TO_RHP_MAPPING`, otherwise refreshing whilst the validate page is open will switch the central pane back to the room.

Reply:
Haha oh I didn't see that you already did this. For some reason when I refresh after tapping `Add bank account` (directly from the settings screen) the chat pane opens in the background.

Reply:
Ah you're right that's an existing bug on staging, we can ignore it here then.